README.md

```diff
@@ -1,2 +1,16 @@
-# terminology
-Codesystems, questionnaires, and validation tools for them, for the All of Us project.
+# All of Us Terminology
+
+Codesystems, questionnaires, and validation tools for them, for the All of Us
+project.
+
+## Usage
+
+![workflow diagram](doc/overview.png)
+
+1.  Export questionnaires from Vibrent's authoring tool and place them in
+    `source/`.
+1.  `tools/validate_and_publish [--commit]` writes to `generated/`.
```

Comment:
I expect a `git push` is required in between `validate_and_publish` and `deploy`?